### ISSUE TITLE
Missing instructions for vtk

### ISSUE THREAD

**Issue by carme-hp** [opendihu/biomesh#8]
I'm trying to install biomesh following the instructions in branch [feature-fiber](https://github.com/opendihu/biomesh/tree/feature-fiber). 

I started with
```
cd biomesh
cmake .
make
```
But I get the following error
>  vtkSmartPointer.h: No such file or directory

I tried fixing it by doing` sudo apt-get install libvtk9-dev`, but the installation was not successful. The error I encounter is

> The following packages have unmet dependencies:
 python3-paraview : Conflicts: python3-vtk9 but 9.1.0+really9.1.0+dfsg2-3build1 is to be installed
E: Error, pkgProblemResolver::Resolve generated breaks, this may be caused by held packages.

**Comment by ghoshgm** [comment]
@carme-hp It would be advisable to build VTK from source. You can follow these instructions.
```
git clone https://github.com/Kitware/VTK.git
mkdir vtk_build && cd vtk_build
cmake .
make -j8
```

**Comment by ghoshgm** [comment]
@carme-hp I would advise to build the dependencies from source or use spack.

**Comment by MakisH** [comment]
> @carme-hp I would advise to build the dependencies from source or use spack.

This should only be a temporary workaround, though, or be documented in detail on what conflicts and for which reason.

**Comment by ghoshgm** [comment]
@MakisH Yes your point is valid. I have mentioned a few points on the README. I agree that a more comprehensive documentation is required.
